doc/sphinx/language/cic.rst

```diff
@@ -336,14 +355,15 @@ following rules.
 .. inference:: Prod-Set
 
    \WTEG{T}{s}
-   s \in \{\Prop, \Set\}
+   s \in \{\SProp, \Prop, \Set\}
    \WTE{\Gamma::(x:T)}{U}{\Set}
    ----------------------------
    \WTEG{\forall~x:T,U}{\Set}
 
 .. inference:: Prod-Type
 
-   \WTEG{T}{\Type(i)}
+   \WTEG{T}{s}
+   s \in \{\SProp, \Type{i}\}
```

Comment:
Did you really mean this change? Isn't it implied by universe cumulativity from the rule above as in the case of `Prop`?

Reply:
No cumulativity with SProp ;)